opentech/static_src/src/app/src/containers/ByStatusListing.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux'
+
+import Listing from '@components/Listing';
+import {
+    getCurrentRound,
+    getCurrentRoundSubmissionsByStatus,
+    getSubmissionsByRoundErrorState,
+    getSubmissionsByRoundLoadingState,
+} from '@selectors/submissions';
+import { setCurrentSubmissionRound, fetchSubmissionsByRound } from '@actions/submissions';
+
+
+class ByStatusListing extends React.Component {
+    componentDidMount() {
+        const { roundId } = this.props;
+        // Update items if round ID is defined.
+        if (roundId !== null && roundId !== undefined) {
+            this.props.loadSubmissions(roundId);
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        const { roundId } = this.props;
+        // Update entries if round ID is changed or is not null.
+        if (roundId !== null && roundId !== undefined && prevProps.roundId !== roundId) {
+            this.props.loadSubmissions(roundId);
+        }
+    }
+
+    render() {
+        const { isLoading, isError } = this.props;
+        return <Listing isLoading={isLoading} isError={isError} items={this.getListingItems()} />;
```

Comment:
I think this component (ByStatusListing) knows too much about the internals of the Listing item.

This is how i see it working:
```javascript
items =  getCurrentRoundSubmissions() //= [{app1},{app2},{app3}]
<Listing items={items} groupBy={"status"} order={['status_1', 'status_2', ...]} />
```

Listing:
```javascript
// Do grouping and ordering then

<ListingGroup status='status1'>
    <ListingItem application={app1}>
    <ListingItem application={app2}>    
</ListingGroup>
<ListingGroup status='status2'>
    <ListingItem application={app3}>    
</ListingGroup>
```